mlflow/sentence_transformers/__init__.py

```diff
@@ -221,6 +221,30 @@ def log_model(
     Log a ``sentence_transformers`` model as an MLflow artifact for the current run.
 
     :param model: A trained ``sentence-transformers`` model.
+
+    .. code-block:: python
+
+        An example of using log_model for a sentence-transformers model and architecture:
+
+        from sentence_transformers import SentenceTransformer
+        import mlflow
+
+        model = SentenceTransformer("all-MiniLM-L6-v2")
+
+        signature = mlflow.models.infer_signature(
+            model_input=example_sentences,
+            model_output=model.encode(example_sentences),
+        )
+
+        with mlflow.start_run():
+            mlflow.sentence_transformers.log_model(
+                model=model,
+                artifact_path="sbert_model",
+                signature=signature,
```

Comment:
We can add the input example too once you have the `example_sentences` variable declared and populated :)